tox.ini

```diff
@@ -8,7 +8,7 @@ commands =
 deps =
     nose
     unittest2
-    protobuf==2.6.0
+    protobuf==3.0.0-alpha1
```

Comment:
We are going to need *some* pin there, maybe `protobuf>=3.0.0-alpha1`.  Ideally, the protobuf team would make a non-alpha release and we could require that.